planet/scripts/cli.py

```diff
@@ -103,3 +107,85 @@ def init(email, password):
     response = call_and_wrap(clientv1().login, email, password)
     write_planet_json({'key': response['api_key']})
     click.echo('initialized')
+
+
+@cli.command('quota')
+@click.option('-k', default=None, prompt=False, hide_input=True, help=(
+    'Enter your API Key'))
```

Comment:
So this one is my fault: I did suggest you add `-k` as an option to this new command "quota". However, since we are already capturing an API key (if the user passes one in) earlier in cli.py, there's no need for us to reinvent the wheel here.

So instead, let's do:
- remove L113-114
- add `k = client_params.get('api_key')` inside the function.

If you're not sure what that does, or why, let me know!

Also, a bit of trivial nagging: this isn't strictly _wrong_, but it's usually nicer (and more "Pythonic") to be explicit with variable names. No reason to use `k` when we mean `key` or `apikey` here.

Reply:
fixed